README.md

```diff
@@ -14,7 +14,15 @@ The filter expression can be any valid python expression that evaluates to `bool
  * `dict`, `list`, `set`, `tuple`
  * `bool`, `chr`, `float`, `int`, `ord`, `str`
  * Any function or symbol from [`math`](https://docs.python.org/3/library/math.html)
+ * Any function from [`statistics`](https://docs.python.org/3/library/statistics.html)
  * Regular expressions via [`re`](https://docs.python.org/3/library/re.html)
+ * custom functions:
+   * `without_na` (keep only values that are not `NA`)
+   * `replace_na` (replace values that are `NA` with some other fixed value)
```

Comment:
At least for `replace_na` a small example invocation is maybe a good idea to include? I guess the other functions should be clear, but this one requires an extra argument. So, just to make it as easy as possible...

Reply:
Good point